engine/src/main/resources/assets/prefabs/player/player.prefab

```diff
@@ -20,6 +20,11 @@
   },
   "VisualCharacter": {
   },
+  // the CapsuleShape is used to detect item entities for pickup. The actual character collider is generated programmatically.
```

Comment:
Hm, I'm wondering whether this additional shape can be derived from the collider (to keep them in sync).
If it is a "special" shape just for item pickup, maybe this should be part of some better-named component?

Reply:
Additional question: where is the character collider shape programmatically set? I see that in https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java#L491-L505 the character collider is created with values taken from the `CharacterMovementComponent`.

How can we keep a single source of truth for the (base) values?

Reply:
I've now moved the pickup-radius information to the CharacterMovementComponent instead, with the same default value, which I think resolves this. There should be no difference in behaviour, but this makes it simpler for other things to alter the pickup radius later.